CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 
 * [#7181](https://github.com/rubocop-hq/rubocop/pull/7181): Sort analyzed file alphabetically. ([@pocke][])
 * [#7188](https://github.com/rubocop-hq/rubocop/pull/7188): Include inspected file location in auto-correction error. ([@pocke][])
+* [#5265](https://github.com/rubocop-hq/rubocop/issues/5265): Improved `Layout/ExtraSpacing` cop to handle nested consecutive assignments. ([@jfelchner][])
```

Comment:
This should go to `master`.

Reply:
@bbatsov not sure what you mean here.  This is under the "Changes" section of the unreleased `master` section.  Do you want it moved somewhere else?

Reply:
Jeff, 0.73 was released after you pushed, so you need to rebase and move the line (with a Changes subheading) under the new master heading.